frontend/src/hooks/api/actions/useApplicationsApi/useApplicationsApi.ts

```diff
@@ -7,22 +7,6 @@ const useApplicationsApi = () => {
 
     const URI = 'api/admin/metrics/applications';
 
-    const storeApplicationMetaData = async (
```

Comment:
removing unused code that was used in application update in the past. today applications cannot be updated from the UI